code/datums/mood_events/drink_events.dm

```diff
@@ -2,6 +2,16 @@
 	mood_change = 3
 	description = "Everything just feels better after a drink or two."
 
+/datum/mood_event/drunk/add_effects(param)
+	// Display blush visual
+	ADD_TRAIT(owner, TRAIT_DRUNK_BLUSHING, SPECIES_TRAIT)
+	owner.update_body()
+
+/datum/mood_event/drunk/remove_effects()
+	// Stop displaying blush visual
+	REMOVE_TRAIT(owner, TRAIT_DRUNK_BLUSHING, SPECIES_TRAIT)
+	owner.update_body()
```

Comment:
The third argument for add and remove trait is the source, and usually it's a descriptor or key related to what added it. 

In this case you're using `SPECIES_TRAIT` which doesn't seem accurate - replace it with something like "DRUNK_TRAIT" or a similar define